File: daySix/Javascript/QuestionThree.js
let moveZeroElementToEndOfArrayIn=(arrayOfNumbers)=>{
    let store = 0;
    
    for(let index = 0; index < arrayOfNumbers.length; index++){
        for(let count = 0; count < arrayOfNumbers.length; count++){
            if(arrayOfNumbers[index] > arrayOfNumbers[count]){
                store = arrayOfNumbers[index];
                arrayOfNumbers[index] = arrayOfNumbers[count];
                arrayOfNumbers[count] = store;
            }
        }
    }
    
    return arrayOfNumbers;
}

arrayOfNumbers = [5,0,3,0,2,0];
console.log(moveZeroElementToEndOfArrayIn(arrayOfNumbers));

File: daySix/python/question_three.py
def moveZeroElementToEndOfArrayIn(array_Of_numbers):
    store = 0
    
    for index in range (len(array_Of_numbers)):
        for count in range (len(array_Of_numbers)):
            if(array_Of_numbers[index] > array_Of_numbers[count]):
                store = array_Of_numbers[index]
                array_Of_numbers[index] = array_Of_numbers[count]
                array_Of_numbers[count] = store
    
    return array_Of_numbers
